### PR TITLE
✅ Update tests with minify Mjml2HtmlOptions

### PR DESCRIPTION
**Pull request by shellyYG**

### CODE REVIEW

File: src/components/template-base.tsx [hunk 1]
```diff
@@ -52,7 +54,18 @@ export const TemplateBase = (props: TemplateBaseProps) => {
               />
               <MjmlText lineHeight="1.5" />
             </MjmlAttributes>
+            <MjmlStyle>
+              {`
+            @media (prefers-color-scheme: dark) {
+                .dark-text {
+                  color: white !important;
+                  }`}
+            </MjmlStyle>
```
Comment (nwingt) [inline src/components/template-base.tsx]: Remove if not used

File: src/templates/transaction.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ export const TransactionTemplate = ({
         paddingBottom={0}
       >
         <MjmlColumn>
-          <MjmlText fontSize={18}>
+          <MjmlText fontSize={18} cssClass={'dark-text'}>
```
Comment (nwingt) [inline src/templates/transaction.tsx]: Remove if not used

File: src/templates/reset-password.tsx [hunk 3]
```diff
@@ -77,7 +77,7 @@ export const ResetPasswordTemplate = ({
         paddingBottom={0}
       >
         <MjmlColumn>
-          <MjmlText fontSize={18}>
+          <MjmlText fontSize={18} color={Colors.Black}>
```
Comment (nwingt) [inline src/templates/reset-password.tsx]: Remove if not used

File: example/index.ts [hunk 4]
```diff
@@ -46,8 +46,8 @@ handleRequest('/reset-password', (req, res) => {
 });
 
 handleRequest('/transaction', (req, res) => {
-  const { subject, body } = getTransactionTemplate({ ...req.query, ...req.body })
-  res.send(subject.concat(body));
+  const { body } = getTransactionTemplate({ ...req.query, ...req.body })
+  res.send(body);
```
Comment (nwingt) [inline example/index.ts]: Let revert this, and open another PR to change all examples

I suggest injecting `subject` prop into `<TemplateBase />`

**Review by nwingt — CHANGES_REQUESTED** [review]

File: example/index.ts (hunk 4 above)
Comment (shellyYG) [reply example/index.ts]: ok, will revert this code

**Review by nwingt — APPROVED** [review]

**Pull request merged**